README.md

```diff
@@ -58,6 +58,28 @@ string_store = StringStore.new
 end
 ```
 
+### Priority
+
+You can specify the priority of the message by using `call_def` and `cast_def` with `priority` argument:
```

Comment:
Am I right that if the original function has `priority` argument - this function will always be prioritized?

Reply:
yeha, that's what I don't like about this :\

Reply:
updated